apps/SageAccountWeb/src/components/TwoFactorAuth/ResetTwoFactorAuth.tsx

```diff
@@ -9,10 +9,8 @@ import {
   useApplicationSessionContext,
   useQuerySearchParam,
 } from 'synapse-react-client'
-import {
-  TwoFactorAuthErrorResponse,
-  TwoFactorAuthResetToken,
-} from '@sage-bionetworks/synapse-types'
+import { TwoFactorAuthResetToken } from '@sage-bionetworks/synapse-types'
+import { TwoFactorAuthErrorResponse } from 'synapse-client/generated/models/TwoFactorAuthErrorResponse'
```

Comment:
All of the error types now come from the new `synapse-client` package, so imports had to be updated everywhere. It's 99% an interface change, so it should be low risk